spark-jobs/src/main/resources/reference.conf

```diff
@@ -95,3 +95,6 @@ timezone="UTC"
 # Optional security settings
 #kafka.security.protocol="SASL_SSL"
 #kafka.sasl.mechanism="GSSAPI"
+
+# Optional - allows original dataframe columns to be dropped
+conformance.allowDataFrameMutability=false
```

Comment:
How about calling it allowOriginalColumnDrop? I'm not so sure about the mutability keyword

Reply:
I would rather keep the naming general. 

What if we do this for other rules? We would have to go with having multiple parameters for each rule like `allowOriginalColumnDrop`, `allowOriginalColumnThisRule`, `allowOriginalColumnThatRule`. Because if we have `allowOriginalColumnDrop`, we cannot replace it just like that unless we increase a major version.

Reply:
I think both comments have merit. I am thinking about the granularity we want to be setting. If indeed the only thing to set is to mutate or not (I'd go with the currently suggested solution of a single field that cover everything). If however, we wanted to be allowing e.g. renaming, dropping, changing columns scheme, and other variants, it would make sense to be more specific. 

But we could always introduce more granularity later.

Adrian has a point that the name should probably contain the word _original_, because it is only the original columns that are (not) restricted from being modified. So my guess is just one field and maybe calling it `allowOriginalColumnsModification` or `allowOriginalColumnsMutability`?

Reply:
I like the `originalColumn`  paired it with `mutability` since that is what we use in our talks